cardano-node/src/Cardano/Tracing/OrphanInstances/Network.hs

```diff
@@ -1351,6 +1507,45 @@ instance ToObject (TracePeerSelection SockAddr) where
     mkObject [ "kind" .= String "ChurnWait"
              , "diffTime" .= toJSON dt
              ]
+  toObject _verb (TraceChurnMode c) =
+    mkObject [ "kind" .= String "ChurnMode"
+             , "event" .= show c ]
+
+instance Aeson.ToJSON AbstractState where
```

Comment:
Comments on what each of the constructors in `AbstractState` means would be useful

Reply:
That's connection manager's abstract state, useful for debugging.  We have a whole chapter about it, and this isn't the right place to provide documentation.  I think a link to the spec should be enough.

Reply:
done